pydash-front/src/authenticated_app/dashboard/statistics_page/VisitorsHeatmapGraph.js

```diff
@@ -0,0 +1,105 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+// Contents:
+import { HeatMap } from '@nivo/heatmap';
+
+// Utils:
+import ResponsiveGraphWrapper from '../../../common/ResponsiveGraphWrapper';
+
+class VisitorsHeatmapGraph extends Component {
+    render() {
+        return (
+            <ResponsiveGraphWrapper height={this.props.height}>
+                <h4>{this.props.title}</h4>
+                <HeatMap
+                    height={this.props.height}
+                    data={this.props.data}
+                    keys={[
```

Comment:
Please don't hardcode this. What about `_.map(_.range(0, 24), num => ""+num+":00")`?